Makefile

```diff
@@ -0,0 +1,80 @@
+DOCTEST = node_modules/.bin/doctest --nodejs '--harmony' --module commonjs --prefix .
+ESLINT = node_modules/.bin/eslint --config node_modules/sanctuary-style/eslint-es3.json --env es3
+ISTANBUL = node_modules/.bin/istanbul
+NPM = npm
+PREDOCTEST = scripts/predoctest
+TRANSCRIBE = node_modules/.bin/transcribe
+XYZ = node_modules/.bin/xyz --repo git@github.com:sanctuary-js/sanctuary-type-classes.git --script scripts/prepublish
+
+TEST = $(shell find test -name '*.js' | sort)
+
+
+.PHONY: all
+all: LICENSE README.md
+
+.PHONY: LICENSE
+LICENSE:
+	cp -- '$@' '$@.orig'
+	sed 's/Copyright (c) .* Sanctuary/Copyright (c) $(shell git log --date=format:%Y --pretty=format:%ad | sort -r | head -n 1) Sanctuary/' '$@.orig' >'$@'
+	rm -- '$@.orig'
+
+README.md: index.js.tmp
+	$(TRANSCRIBE) \
+	  --heading-level 4 \
+	  --url 'https://github.com/sanctuary-js/sanctuary-type-classes/blob/v$(VERSION)/index.js#L{line}' \
+	  -- '$<' \
+	| LC_ALL=C sed 's/<h4 name="\(.*\)#\(.*\)">\(.*\)\1#\2/<h4 name="\1.prototype.\2">\3\1#\2/' >'$@'
+
+.INTERMEDIATE: index.js.tmp
+index.js.tmp: index.js
+	sed -e '/^[/][/]:/ s,\([A-Z][^ ]*\),<a href="#\1">\1</a>,g' -e 's,^//:,//.,' '$<' >'$@'
```

Comment:
This line is neat. It turns all the names in the ASCII type-class graph into links to their respective sections. We get the benefits of fantasyland/fantasy-land#160 without adulterating the graph in __index.js__. :grinning: